edx_proctoring/templates/proctoring/seq_proctored_exam_error.html

```diff
@@ -0,0 +1,56 @@
+{% load i18n %}
+<div class="failure sequence proctored-exam" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      Your proctoring session is in error
+    {% endblocktrans %}
+  </h3>
+
+  <h4>
+    {% blocktrans %}
+      Your Proctoring Session review: <b class="failure"> Failed </b>
+    {% endblocktrans %}
+  </h4>
+
+  <p>
+    {% blocktrans %}
+      It appears that your proctoring session has been shut down while you were taking this
```

Comment:
"Your proctoring session ended before you completed this exam, so your proctoring results are incomplete. You will not be eligible to purchase academic credit for this course."